tests/BrokerageClientTest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+class ExchangeClientTest extends \PHPUnit\Framework\TestCase {
+    public function testInstantiates() {
+        $cfx = new \CFX\SDK\Brokerage\Client('https://null.cfxtrading.com', '12345', 'abcde', new \GuzzleHttp\Client());
+        $this->assertInstanceOf("\\CFX\\SDK\\ClientInterface", $cfx);
+    }
+
+
+    // Subclients
+
+    public function testCanGetAssetsSubclient() {
+        $cfx = new \CFX\SDK\Brokerage\Client('https://null.cfxtrading.com', '12345', 'abcde', new \GuzzleHttp\Client());
+        $this->assertInstanceOf('\\CFX\\SDK\\SubclientInterface', $cfx->assets);
```

Comment:
I know I set a bad model here when I did my Exchange tests, but you should change these to assert instance of `\CFX\SDK\Brokerage\[whatever client interface]` instead of the more general `SubclientInterface`. My bad.